hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ResponseTemplate.java

```diff
@@ -146,4 +146,12 @@ void Exception(int errorCode, String className, @Nullable String message, StackT
      */
     @Response(ResponseMessageConst.READ_RESULT_SET)
     void ReadResultSet(int readCount, List<Data> items);
+
+    /***
+     * @param tableIndex the last tableIndex processed
```

Comment:
Will the user use this tableIndex in the next call to the iterate method? If so, let's put this info.